applications/app/controllers/InteractiveController.scala

```diff
@@ -105,51 +105,37 @@ class InteractiveController(
   override def canRender(i: ItemResponse): Boolean = i.content.exists(_.isInteractive)
 
   override def renderItem(path: String)(implicit request: RequestHeader): Future[Result] = {
-    ApplicationsDotcomRenderingInterface.getRenderingTier(path) match {
-      case Legacy => {
+    // See comment id: DF38D2B4-614D for why we have two rendering.
+    ApplicationsInteractiveRendering.getRenderingTier(path) match {
+      case Regular => {
         lookup(path) map {
           case Left(model)  => render(model)
           case Right(other) => RenderOtherStatus(other)
         }
       }
-      case Election2020Hack => renderInteractivePageElection2020_v2(path)
-      case DotcomRendering => {
-        val remoteRenderer = DotcomRenderingService()
-        val range = ArticleBlocks
-
-        val path2 = "world/2013/jun/09/edward-snowden-nsa-whistleblower-surveillance"
-        capiLookup
-          .lookup(path2, Some(range))
-          .map(responseToModelOrResult)
-          .recover(convertApiExceptions) // Future[Either[(ArticlePage, Blocks), Result]]
-          .flatMap { e =>
-            e match {
-              case Left((article, blocks)) => {
-                val pageType: PageType = PageType(article, request, context)
-                remoteRenderer.getAMPArticle(wsClient, article, blocks, pageType)
-              }
-              case Right(other) => Future.successful(Ok("case: ade30b6a-de4a-469d-8ad8-701996e5be06"))
-            }
-          }
-
-        // val html: String = ApplicationsDotcomRenderingInterface.getHtmlFromDCR()
-        // Future.successful(Ok(html))
-      }
+      case USElection2020AmpPage => renderInteractivePageUSPresidentialElection2020_v2(path)
     }
   }
 
   // ---------------------------------------------
-  // Election2020
+  // US Presidential Election 2020
+
+  /*
+    The two following functions implement the rendering of the US Election 2020 Election Tracker Amp Page.
+    Only the second version is used, but both are kept for historical interest. Notably, the first version has the code
+    needed to expand that logic to all interactives (if we wanted to do that one day, before DCR takes that kind of
+    rendering over).
```

Comment:
I'd probably err towards just removing the v1 rather than keeping it around for history. I think we could easily bring something like that back even with it no longer in the code base, and really we ought to have it rendering through DCR as a next step? (which would mean all of this is redundant?)

Reply:
Done.